android/KMEA/app/src/main/java/com/tavultesoft/kmea/data/CloudRepository.java

```diff
@@ -258,14 +272,15 @@ private void preCacheDataSet(@NonNull Context context, UpdateHandler updateHandl
     // Consolidate kmp.json info from packages/
     JSONObject kmpLanguagesArray = wrapKmpKeyboardJSON(JSONUtils.getLanguages());
     JSONArray kmpLexicalModelsArray = JSONUtils.getLexicalModels();
+    boolean fromKMP = true;
```

Comment:
```suggestion
    final boolean fromKMP = true;
```

If this never changes, make it clear that it's a constant :)